drivers/led/lp5569.c

```diff
@@ -318,9 +327,12 @@ static DEVICE_API(led, lp5569_led_api) = {
 		.bus = I2C_DT_SPEC_INST_GET(id),                                                   \
 		.enable_gpio = GPIO_DT_SPEC_INST_GET_OR(id, enable_gpios, {0}),                    \
 		.cp_mode = DT_ENUM_IDX(DT_DRV_INST(id), charge_pump_mode),                         \
+		.int_clk_en = DT_INST_PROP_OR(id, int_clk_en, false),                              \
+		.cp_return_1x = DT_INST_PROP_OR(id, cp_return_1x, false),                          \
+		.powersave_en = DT_INST_PROP_OR(id, powersave_en, false),                          \
```

Comment:
Did it not work with power save enabled?

Reply:
Problem was that with `cp-mode="auto"` and `powersave-en` then the engine could stop running.
Also the `powersave-en` is default disabled, so there was some conflicts there.

Now we should be able to fully control it and have expected default behavior :+1: